ci/install_abseil.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+export DEBIAN_FRONTEND=noninteractive
+
+if ! type cmake > /dev/null; then
+    #cmake not installed, exiting
+    exit 1
+fi
+export BUILD_DIR=/tmp/
```

Comment:
Is `export` necessary here? Does shell variable work?

Reply:
No, it's not required. Made it a shell variable.